### DISCUSSION TITLE
Smaller icon button options

### DISCUSSION THREAD

**Discussion by SvdSinner** [ant-design-blazor/ant-design-blazor#3276] [Ideas]
I've been working on a proof of concept of switching our Blazor front end to AntBlazor.  The desired iconography of the front end uses several small icon buttons to give the user a consistent user experience where users can get more details of common objects.  (Example:  Phone numbers will have a small ellipsis icon button at the end to allow the user to switch from a small format like '(523)555-1234' to 'Main: +1(523)555-1234 x8554' if they need to see that additional level of detail)

The problem is that even with the smallest settings I have tried:
`<Button Size="small" Shape="@ButtonShape.CircleOutline" Icon="@IconType.Outline.Plus" />`<Button Size=@ButtonSize.Small Shape="@ButtonShape.CircleOutline" Icon="@IconType.Outline.Plus" />`
the icon button is still about twice the desired size.

Are there better settings to get smaller icon buttons?  (NOTE:  I want to make these adjustments inside of the Ant-Design-Blazor way of doing things with as little custom CSS as possible)  Or would I need an an XSmall ButtonSize and do a pull request to have it added?

**Comment by ElderJames** [comment]
Thank you for suggestion @SvdSinner . Pull requests are always welcome!
